draft-irtf-cfrg-vdaf.md

```diff
@@ -3355,20 +3368,29 @@ def eval(IdpfPoplar, agg_id, public_share, init_seed,
             # wasteful. Implementations can eliminate this added
             # complexity by caching nodes (i.e., `(seed, ctrl)`
             # pairs) output by previous calls to `eval_next()`.
-            (seed, ctrl, y) = IdpfPoplar.eval_next(seed, ctrl,
-                correction_words[current_level], current_level, bit, binder)
+            (seed, ctrl, y) = IdpfPoplar.eval_next(
+                seed,
+                ctrl,
+                correction_words[current_level],
+                current_level,
+                bit,
+                binder,
+            )
         out_share.append(y if agg_id == 0 else vec_neg(y))
     return out_share
 
-# Compute the next node in the IDPF tree along the path determined by
-# a candidate prefix. The next node is determined by `bit`, the bit
-# of the prefix corresponding to the next level of the tree.
-#
-# TODO Consider implementing some version of the optimization
-# discussed at the end of [BBCGGI21, Appendix C.2]. This could on
-# average reduce the number of AES calls by a constant factor.
 def eval_next(IdpfPoplar, prev_seed, prev_ctrl,
               correction_word, level, bit, binder):
+    """
+    Compute the next node in the IDPF tree along the path determined by
+    a candidate prefix. The next node is determined by `bit`, the bit of
+    the prefix corresponding to the next level of the tree.
+
+    TODO Consider implementing some version of the optimization
+    discussed at the end of [BBCGGI21, Appendix C.2]. This could on
+    average reduce the number of AES calls by a constant factor.
+    """
+
```

Comment:
Consistency: Do we prefer a blankline after docucomment or not?

Reply:
I'm not aware of a norm regarding this, I looked around in the `requests` codebase as an example and saw a combination. I tried to include a blank line when it helped with clarity, such as when a line comment immediately followed, or the function was big enough that it already had some blank lines. I think this gets treated as a special case of blank lines within a function/method body (as the docstring is technically a statement containing a string literal expression) and left up to the author's discretion.